java-client/src/main/java/com/cumulocity/sdk/client/RestConnector.java

```diff
@@ -201,6 +201,15 @@ public <T extends ResourceRepresentation> T putStream(String path, MediaType med
         return parseResponseWithoutId(responseClass, builder.put(stream), OK.getStatusCode());
     }
 
+    @Override
+    public void postStreamWithoutResponse(String path, InputStream inputStream, MediaType inputStreamMediaType) {
+        Builder builder = getResourceBuilder(path);
```

Comment:
**[nitpick]** I would like to be consistent (just like you did for Response object) as I believe in using final liberally :) Feel free to disregard it though.

Reply:
final removed completely to be consistent with other methods :)